src/main/java/easton/view/Ui.java

```diff
@@ -0,0 +1,60 @@
+package easton.view;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui<T> {
```

Comment:
Why does this class have to be a generic type?

Reply:
Well I agree it is not necessary because of the scale of this ip, however, when I was planning out the design I thought it would also make sense to make this class generic so that if there were other Classes we would want to display (e.g. listing) I would not need to write code. 

However, I would appreciate your input on this thought of my @weijianwong. Thank you!